### DISCUSSION TITLE
Como adicionar um novo projeto?

### DISCUSSION THREAD

**Discussion by rodrigoslayertech** [github/brasil#12] [General]
Olá mundo! 

Sou Rodrigo Vieira e tenho um projeto chamado [bootgly/bootgly](https://github.com/bootgly/bootgly)!

Quais são as regras para adicionar no site que listas os projetos e pessoas mantenedoras?

Grato!

**Comment by AndreaGriffiths11** [comment]
Hola Rodrigo, apologies for the delay here! Check out our contributing guide if you still need to add. TYSM for being here ❤️https://github.com/github/brasil/blob/main/CONTRIBUTING.md
